languages/javascript/reference/keywords/const.md

```diff
@@ -1 +1,26 @@
 # `const`
+
+The `const` keyword is used to create a read-only reference to a value.
+```javascript
+const NAME = "Bob";
+```
+The value it holds is still [mutable][concept-immutability], but the variable identifier cannot be reassigned:
+```javascript
+const NAME = "Bob";
+
+NAME = "Steve";
+// TypeError: Assignment to constant variable
+```
+or redeclared:
+```javascript
+const NAME = "Bob"
+
+const NAME = "Sally"
+// SyntaxError: Identifier 'NAME' has already been declared
+```
+Variables declared using the `const` keyword are [block-scoped][concept-scope], meaning they are only accessible within the current block, and any contained sub-blocks.
+
+The value that a variable declared using `const` holds can only be referenced *after* it has been defined. This is known as the [Temporal Dead Zone][concept-scope#temporal-dead-zone].
```

Comment:
https://github.com/exercism/v3/pull/285/files?short_path=120c2fc#diff-120c2fc08d0d75857fc6f2cddfa982ae

That doesn't work. It will work if you create a separate link reference.